client/src/auth/logout/LogoutSuccess.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import MuiAlert from "@material-ui/lab/Alert";
+
+export type LogoutSuccessProps = {
+  username: string;
+  secondsTillRedirect: number;
+};
+
+export function LogoutSuccess(props: LogoutSuccessProps) {
+  return (
+    <>
+      <MuiAlert elevation={6} variant="filled" severity="success">
```

Comment:
Könnte man diesen Alert nicht auch mit Hilfe der Snackbar lösen?
Der Vorteil wäre dann, dass alle Benachrichtigungen gleich aussehen.

Reply:
Doch aber, die Snackbar ist glaube ich noch nicht in diesem Branch 😃